### PR TITLE
Exit save menu using controller

### LINKED ISSUE #1173: AE OG Bug - Save screen cannot be quit with controller
**Description of issue:** 
The save screen in the pause menu cannot be quit with any of the usual controller inputs, pressing <kbd>Esc</kbd> on the keyboard is necessary to get out.

Expected behavior would be to exit with Y/Triangle or Start.

Reportedly OG Oddysee lets you get out of this screen by pressing Start or Select on the controller, but I haven't been able to verify this with my virtual XInput controller, so this would need fixing on the Oddysee side as well.

**Steps to repro:** 
1. Set the controller as the active input device
2. Start a new game
3. Pause the game, select *Save*
4. Try quitting with any of the controller buttons

### PR DESCRIPTION
**Pull request by deanff**
Fixes #1173

### CODE REVIEW

File: Source/AliveLibAE/Sys.cpp [hunk 1]
```diff
@@ -29,6 +29,15 @@ ALIVE_VAR(1, 0xBBB9EC, HINSTANCE, sInstance_BBB9EC, nullptr);
 ALIVE_VAR(1, 0xBBB9FC, int, sCmdShow_BBB9FC, 0);
 ALIVE_VAR(1, 0xBBFB04, TWindowHandleType, hWnd_BBFB04, nullptr);
 
+#if ORIGINAL_PS1_BEHAVIOR
+bool saveMenuOpen = false;
```
Comment (paulsapps) [inline Source/AliveLibAE/Sys.cpp:33]: this can be static

File: Source/AliveLibAO/PauseMenu.cpp [hunk 2]
```diff
@@ -384,6 +392,24 @@ void PauseMenu::VUpdate_44DFB0()
 
                     auto last_pressed = Input_GetLastPressedKey_44F2C0();
                     char lastPressedKeyNT[2] = { last_pressed, 0 };
+
+#if ORIGINAL_PS1_BEHAVIOR // OG Change - Exit save menu using controller
+                    if (last_pressed == VK_ESCAPE || last_pressed == VK_RETURN) // Keyboard ESC or ENTER
```
Comment (paulsapps) [inline Source/AliveLibAO/PauseMenu.cpp:397]: this whole block is duplicated earlier - should prob be in its own func

Comment (deanff) [reply Source/AliveLibAO/PauseMenu.cpp:397]: Should I make the function in AE or AO?

Comment (paulsapps) [reply Source/AliveLibAO/PauseMenu.cpp:397]: Oh yeah I forgot everything exists twice - yeah just leave it copy pasted for now I guess

**Pull request merged**